### ISSUE TITLE
Can I add extra columns with special names that contain additional data stream content for the ingest?

### ISSUE THREAD

**Issue by McFateM** [mnylc/islandora_multi_importer#18]
Case in point...we frequently generate multi-page PDFs for ingest from TIFF images and wish to save those original TIFFs as archival datastreams in the Fedora object.  It would be great if I could identify or label a column in the CSV, maybe name it DS:ARCHIVE001 and have the system recognize that that is to be imported as a supplemental ARCHIVE001 datastream in the Fedora object.

**Comment by patdunlavey** [comment]
New datastreams must have a mimetype defined. Perhaps this could be part of the naming convention?

E.g. `DS:ARCHIVE001:image/tiff`

Implementing this may not be trivial since, among other things, it will require creating a different kind of behavior in the mappings form.

**Comment by DiegoPino** [comment]
@McFateM @patdunlavey I wonder if this use case is to allow, out of the CMODEL blueprint definition, extra DSID to be created?

**Comment by McFateM** [comment]
Yes, that is essentially what I had in mind, but it’s less of an issue now.  At one time we proposed adding an archive-quality datastream to some content, like TIFF images of book pages that were ingested from PDF.  However, we are thinking now we’ll back down from that and just insert those TIFFs into the bags that we generate for backup/cold storage.  That way the TIFFs won’t be a burden for Fedora, but we’ll still have them as archival copies as intended.

Thanks.

-Mark M.
